benchmarks/benchmarks/sperf/combine_regions.py

```diff
@@ -165,6 +169,9 @@ def recombine(self):
         )
         return result
 
+    def save(self):
+        save(self.recombined_cube, self.temp_save_path)
```

Comment:
I see you have two different function/methods called`save` here. It doesn't look like this is causing a namespace issue but it does make reading a bit more difficult. Maybe rename this method to something like `save_cube` or `save_recombined_cube`?

Reply:
ab467b5f5bd0ab9fbdef39c14d46dba6ff73d60e